<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>558</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>739</width>
    <height>558</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>739</width>
    <height>558</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>739</width>
    <height>558</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SysAdminHelper</string>
  </property>
  <widget class="QPushButton" name="pb_ipconfig">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold;
</string>
   </property>
   <property name="text">
    <string>ipconfig</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_tracer">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>80</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold;</string>
   </property>
   <property name="text">
    <string>tracer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: Bold;</string>
   </property>
   <property name="text">
    <string>pathping</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_ping">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>190</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ping</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>getmac</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>444</x>
     <y>310</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>nslookup</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>284</x>
     <y>440</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>netstat</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
